README.rst

```diff
@@ -562,10 +562,26 @@ ensuring also that ``openwisp_network_topology`` has been removed:
 
     INSTALLED_APPS = [
         # ... other apps ...
-        # 'openwisp_network_topology'  <-- comment out or delete this line
-        'sample_network_topology'
+        'openwisp_utils.admin_theme',
+        # all-auth
+        'django.contrib.sites',
+        'allauth',
+        'allauth.account',
+        'allauth.socialaccount',
+        # controller
+        'openwisp_controller.pki',
+        'openwisp_controller.config',
+        'reversion',
+        'sortedm2m',
+        # network topology
+        # 'sample_network_topology' <-- uncomment and replace with your app-name here
         # optional, only if you need to extend the integration app
-        'sample_integration_device'
+        # 'sample_integration_device' <-- uncomment and replace with your integration-app-name here
```

Comment:
@nemesisdesign I see.
If it is not inherently clear that these lines are not suppose to be commented then I think it'll cause more confusion than benefit, so I have instead done as asked in the first review and commented all of the lines.

Reply:
ok, I think it's fine.
I think it's the user responsibility to carefully read the hints we put here.

We can try to make the hints more evident, like you have done with the arrows here, but leaving broken code on purpose could be counterproductive: most people nowadays expect examples to work if they copy&paste and if the code they copy breaks they'll be annoyed.